libpod/container_attach.go

```diff
@@ -81,11 +85,19 @@ func (c *Container) attachContainerSocket(resize <-chan remotecommand.TerminalSi
 			logrus.Warnf("Failed to write to control file to resize terminal: %v", err)
 		}
 	})
-	logrus.Debug("connecting to socket ", c.AttachSocketPath())
 
-	conn, err := net.DialUnix("unixpacket", nil, &net.UnixAddr{Name: c.AttachSocketPath(), Net: "unixpacket"})
+	socketPath := c.AttachSocketPath()
+
+	maxUnixLength := int(C.unix_path_length())
+	if maxUnixLength < len(socketPath)-1 {
+		socketPath = socketPath[0:maxUnixLength]
+	}
+
+	logrus.Debug("connecting to socket ", socketPath)
```

Comment:
I think that should be `Debugf("... %v", socketPath)`.

Reply:
+1, I'm surprised gofmt or govet didn't ping on this.